modules/uploading-an-existing-notebook-file-from-a-git-repository-using-code-server.adoc

```diff
@@ -0,0 +1,30 @@
+:_module-type: PROCEDURE
+
+[id='uploading-an-existing-notebook-file-from-a-git-repository-using-code-server_{context}']
+= Uploading an existing notebook file from a Git repository by using code-server
+
+[role='_abstract']
+You can use the code-server user interface to clone a Git repository into your workspace to continue your work or integrate files from an external project.
+
+.Prerequisites
+* You have a running code-server workbench.
+* You have read access for the Git repository you want to clone.
+
+.Procedure
+. Copy the HTTPS URL for the Git repository.
++
+** In GitHub, click *&#10515; Code* -> *HTTPS* and click the *Copy URL to clipboard* icon.
```

Comment:
global comment:
 add "then" after "and" when combining two steps in one:
For example:
In GitHub, click *&#10515; Code* -> *HTTPS* and click the *Copy URL to clipboard* icon.
change to:
In GitHub, click *&#10515; Code* -> *HTTPS* and then click the *Copy URL to clipboard* icon.